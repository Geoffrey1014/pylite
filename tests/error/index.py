x = [1, 2, 3]
print 'Start'
print x[4] 
print 'End'
