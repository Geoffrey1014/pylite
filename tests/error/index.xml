lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    new_list 3
    load_const c0
    load_const c1
    dup_top 2
    store_item
    load_const c2
    load_const c0
    dup_top 2
    store_item
    load_const c3
    load_const c2
    dup_top 2
    store_item
    store_global c4
    load_const c5
    print
    load_const c6
    load_global c4
    load_item
    print
    load_const c7
    print
    load_const c8
    return
  const_vector
    1.000000
    0.000000
    2.000000
    3.000000
    x
    "Start"
    4.000000
    "End"
    None
  guard_vector
  line_number_map 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 2 3 3 3 3 4 4 1 1
