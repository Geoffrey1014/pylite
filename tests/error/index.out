Start
IndexError:
  At index.py, line 3, in __init__()
